xml/article_setup.xml

```diff
@@ -1319,12 +1321,12 @@ rmdir /dev/cpuset/$1
      <para>
       The real-time I/O class always gets the highest priority for disk access.
       The other schedulers will only be served if no real-time request is
-      present. This scheduling class may easily lock up the system if not
+      present. This scheduling class may easily lock the system if not
```

Comment:
Are you sure about this one? To me, "lock" sounds like sth. a user does intentionally (i.e. screen lock), "lock up" sounds like sth. a computer does to itself when it freaks out.

You also did not change the term on l.553 (which I think is a good decision).

Reply:
OK, good point. Reverted.